lib/mayaUsd/ufe/UsdAttributes.cpp

```diff
@@ -332,21 +332,40 @@ bool UsdAttributes::canAddAttribute(
 {
     // See if we can edit this attribute, and that it is not already part of the schema or node
     // definition
-    if (!item || !item->prim().IsActive() || UsdAttributes(item).hasAttribute(name)) {
```

Comment:
This feels very tricky to me.  The previous meaning of the function was: "Can I add an attribute with this exact name?"

And previously, the answer was: if an attribute of that name already exists, no you can't.  You've changed that answer to yes you can, but it won't be an attribute of that name.  In fact, the name argument is completely unused now.

Either we change the interface and the documentation of the function, or we revert this change and create a separate function that ignores the name.

Reply:
I think it's better to change the interface and update the documentation since this function is only used once by the `UsdAddAttributeCommand::create` function.